docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '3.8'
+
+services:
+    db:
+        image: postgres
+        restart: always
+        ports:
+            - 5432:5432
+        environment:
+            - POSTGRES_USER=botlistbot
+            - POSTGRES_DB=botlistbot
+            - POSTGRES_PASSWORD=botlistbot
+        volumes:
+            - db:/var/lib/postgresql/data
+
+volumes:
+    db:
```

Comment:
If there's no volume, doesn't that mean all data is lost every time you docker up?

Reply:
To create a volume, you just need the name and then you specify it in the service.